docs/src/app/components/pages/components/app-bar.jsx

```diff
@@ -21,6 +22,11 @@ export default class AppBarPage extends React.Component {
     return (
       <div>
         <MarkdownElement text={appBarReadmeText} />
+        <RaisedButton
+          label="View Waterfall example"
+          linkButton={true}
+          href="/#/examples/app-bar-waterfall"
+          primary={true} />
```

Comment:
I think putting this after the `CodeExample`s is more pleasing to the eye, don't you think @oliviertassinari ?